app/(technician)/technician/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import MenuButton from "@/components/navigation/MenuButton";
+import NavigationBar from "@/components/navigation/NavigationBar";
+import { Metadata } from "next";
+import { ReactNode } from "react";
+
+type Props = {
+  statistics: ReactNode,
+  recent: ReactNode
+}
+
+export const metadata: Metadata = {
+  title: "Technician | Ovoflow"
+}
+
+export default function FarmerDashboard({ statistics, recent }: Props) {
```

Comment:
```suggestion
export default function TechnicianDashboard({ statistics, recent }: Props) {
```

Farmer Dashboard is a little confusing here.